---
title: "Single-nucleus transcriptome analysis reveals cell-type-specific molecular signatures across reward circuitry in the human brain"
authors:

- "Matthew N. Tran __*__"
- "Kristen R. Maynard __*__"
- "Abby Spangler"
- "Louise A. Huuki"
- "Kelsey D. Montgomery"
- "Vijay Sadashivaiah"
- "Madhavi Tippani"
- "Brianna K. Barry"
- "Dana B. Hancock"
- "Stephanie C. Hicks"
- "Joel E. Kleinman"
- "Thomas M. Hyde"
- "admin"
- "Andrew E. Jaffe &dagger;"
- "Keri Martinowich &dagger;"

date: "2021-10-06T00:00:00Z"
doi: "10.1016/j.neuron.2021.09.001"

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Neuron*"
publication_short: ""

abstract: "Single-cell gene expression technologies are powerful tools to study cell types in the human brain, but efforts have largely focused on cortical brain regions. We therefore created a single-nucleus RNA-sequencing resource of 70,615 high-quality nuclei to generate a molecular taxonomy of cell types across five human brain regions that serve as key nodes of the human brain reward circuitry: nucleus accumbens, amygdala, subgenual anterior cingulate cortex, hippocampus, and dorsolateral prefrontal cortex. We first identified novel subpopulations of interneurons and medium spiny neurons (MSNs) in the nucleus accumbens and further characterized robust GABAergic inhibitory cell populations in the amygdala. Joint analyses across the 107 reported cell classes revealed cell-type substructure and unique patterns of transcriptomic dynamics. We identified discrete subpopulations of D1- and D2-expressing MSNs in the nucleus accumbens to which we mapped cell-type-specific enrichment for genetic risk associated with both psychiatric disease and addiction."

# Summary. An optional shortened abstract.
summary:

tags:
- snRNA-seq
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: 'https://doi.org/10.1016/j.neuron.2021.09.001'
url_code: 'https://github.com/LieberInstitute/10xPilot_snRNAseq-human'
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''
url_preprint: 'https://www.biorxiv.org/content/10.1101/2020.10.07.329839v1'

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**bioRxiv**](https://doi.org/10.1101/2020.10.07.329839)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

<!--

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Click the *Slides* button above to demo Academic's Markdown slides feature.
{{% /callout %}}

Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
-->

{{% tweet user="mattntran" id="1442509542412128258"%}}
{{% tweet user="mattntran" id="1314323646815629312"%}}
