+++
url_code = "https://figshare.com/articles/Scripts_outputs_for_paper_Human_splicing_diversity_across_the_Sequence_Read_Archive_by_Nellore_et_al_/3811629/1"
authors = ["A Nellore", "AE Jaffe", "JP Fortin", "J Alquicira-Hern√°ndez", "__L Collado-Torres__", "S Wang", "RA Phillips", "N Karbhari", "KD Hansen", "B Langmead", "JT Leek"]
url_project = "http://intropolis.rail.bio/"
image = ""
image_preview = ""
publication_types = ["0"]
url_pdf = "http://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-1118-6?site=genomebiology.biomedcentral.com"
abstract = "Background: Gene annotations, such as those in GENCODE, are derived primarily from alignments of spliced cDNA sequences and protein sequences. The impact of RNA-seq data on annotation has been confined to major projects like ENCODE and Illumina Body Map 2.0. Results: We aligned 21,504 Illumina-sequenced human RNA-seq samples from the Sequence Read Archive (SRA) to the human genome and compared detected exon-exon junctions with junctions in several recent gene annotations. We found 56,861 junctions (18.6%) in at least 1000 samples that were not annotated, and their expression associated with tissue type. Junctions well expressed in individual samples tended to be annotated. Newer samples contributed few novel well-supported junctions, with the vast majority of detected junctions present in samples before 2013. We compiled junction data into a resource called intropolis available at http://intropolis.rail.bio. We used this resource to search for a recently validated isoform of the ALK gene and characterized the potential functional implications of unannotated junctions with publicly available TRAP-seq data. Conclusions: Considering only the variation contained in annotation may suffice if an investigator is interested only in well-expressed transcript isoforms. However, genes that are not generally well expressed and nonetheless present in a small but significant number of samples in the SRA are likelier to be incompletely annotated. The rate at which evidence for novel junctions has been added to the SRA has tapered dramatically, even to the point of an asymptote. Now is perhaps an appropriate time to update incomplete annotations to include splicing present in the now-stable snapshot provided by the SRA."
publication = "Genome Biology"
selected = false
url_slides = ""
highlight = true
title = "Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive"
url_dataset = "https://figshare.com/articles/intropolis_exon-exon_junctions_across_publicly_available_RNA-seq_samples_on_the_Sequence_Read_Archive/3811680/1"
url_video = ""
date = "2016-12-30T17:44:49-05:00"
publication_short = ""
math = false
abstract_short = "We found 56,861 junctions (18.6%) in at least 1000 samples that were not annotated out of 21,504 samples, and their expression associated with tissue type. We compiled junction data into a resource called intropolis available at http://intropolis.rail.bio."

[[url_custom]]
    name = "Pre-print"
    url = "http://biorxiv.org/content/biorxiv/early/2016/01/29/038224"
+++

