---
title: "Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex"
authors:
- "Kristen R Maynard __*__"
- "[__Leonardo Collado-Torres__](/authors/admin) __*__"
- "Lukas M. Weber"
- "Cedric Uytingco"
- "Brianna K. Barry"
- "Stephen R. Williams"
- "Joseph L. Catallini II"
- "Matthew N. Tran"
- "Zachary Besich"
- "Madhavi Tippani"
- "Jennifer Chew"
- "Yifeng Yin"
- "Joel E. Kleinman"
- "Thomas M. Hyde"
- "Nikhil Rao"
- "Stephanie C. Hicks"
- "Keri Martinowich &dagger;"
- "Andrew E Jaffe &dagger;"


date: "2021-02-08T00:00:00Z"
doi: "10.1038/s41593-020-00787-0"

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Nature Neuroscience*"
publication_short: ""

abstract: We used the 10x Genomics Visium platform to define the spatial topography of gene expression in the six-layered human dorsolateral prefrontal cortex. We identified extensive layer-enriched expression signatures and refined associations to previous laminar markers. We overlaid our laminar expression signatures on large-scale single nucleus RNA-sequencing data, enhancing spatial annotation of expression-driven clusters. By integrating neuropsychiatric disorder gene sets, we showed differential layer-enriched expression of genes associated with schizophrenia and autism spectrum disorder, highlighting the clinical relevance of spatially defined expression. We then developed a data-driven framework to define unsupervised clusters in spatial transcriptomics data, which can be applied to other tissues or brain regions in which morphological architecture is not as well defined as cortical laminae. Last, we created a web application for the scientific community to explore these raw and summarized data to augment ongoing neuroscience and spatial transcriptomics research (http://research.libd.org/spatialLIBD).

# Summary. An optional shortened abstract.
summary:

tags:
- spatial
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: https://doi.org/10.1038/s41593-020-00787-0
url_code: https://bioconductor.org/packages/spatialLIBD
url_dataset: https://github.com/LieberInstitute/HumanPilot
url_poster: ''
url_project: ''
url_slides: ''
url_source: https://github.com/LieberInstitute/spatialLIBD
url_video: ''
url_preprint: 'https://www.biorxiv.org/content/10.1101/2020.02.28.969931v1'

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**bioRxiv**](https://doi.org/10.1101/2020.02.28.969931)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: ["spatial"]

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

<!--

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Click the *Slides* button above to demo Academic's Markdown slides feature.
{{% /callout %}}

Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
-->

{{% tweet user="andrewejaffe" id="1358825640686989313" %}}
{{% tweet user="lcolladotor" id="1233661576433061888" %}}
