+++
title = "Reproducible RNA-seq analysis using recount2"
date = "2017-04-11T16:45:43-05:00"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["[__L Collado-Torres__](/authors/admin) __*__", "A Nellore __*__", "K Kammers", "SE Ellis", "MA Taub", "KD Hansen", "AE Jaffe", "B Langmead", "JT Leek"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "Nature Biotechnology"
publication_short = ""

# Abstract.
abstract = "recount2 is a resource of processed and summarized expression data spanning over 70,000 human RNA-seq samples from the Sequence Read Archive (SRA). The associated recount Bioconductor package provides a convenient API for querying, downloading, and analyzing the data. Each processed study consists of meta/phenotype data, the expression levels of genes and their underlying exons and splice junctions, and corresponding genomic annotation. We also provide data summarization types for quantifying novel transcribed sequence including base-resolution coverage and potentially unannotated splice junctions. We present workflows illustrating how to use recount to perform differential expression analysis including meta-analysis, annotation-free base-level analysis, and replication of smaller studies using data from larger studies. recount provides a valuable and user-friendly resource of processed RNA-seq datasets to draw additional biological insights from existing public data. The resource is available at https://jhubiostatistics.shinyapps.io/recount/."

# Summary. An optional shortened abstract.
summary = ""

# Digital Object Identifier (DOI)
doi = "10.1038/nbt.3838"

# Is this a featured publication? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["recount2"]

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["recount2"]

# Links (optional).
url_pdf = "http://www.nature.com/nbt/journal/v35/n4/full/nbt.3838.html"
url_preprint = "https://www.biorxiv.org/content/10.1101/068478v1"
url_code = "https://github.com/leekgroup/recount-website"
url_dataset = ""
url_project = "http://leekgroup.github.io/recount-analyses/"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
links = [{name = "Bioconductor", url = "http://bioconductor.org/packages/recount"}]

# Does this page contain LaTeX math? (true/false)
math = true

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Image credit: [*Nature Biotechnology**](https://doi.org/10.1038/nbt.3838)"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "bottom"
+++

<!-- More detail can easily be written here using *Markdown* and $\rm \LaTeX$ math code. -->
