+++
title = "recount workshop: Learn to leverage 70,000 human RNA-seq samples for your projects"

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date = 2017-07-27T13:30:00
date_end = 2017-07-27T13:30:00
all_day = false

# Schedule page publish date (NOT talk date).
publishDate = ""

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["admin"]

# Location of event.
location = "Boston, MA, US"

# Name of event and optional event URL.
event = "BioC2017"
event_url = "http://bioconductor.org/help/course-materials/2017/BioC2017/"

# Abstract. What's your talk about?
abstract = "The recount2 project re-processed RNA sequencing (RNA-seq) data on over 70,000 human RNA-seq samples spanning a variety of tissues, cell types and disease conditions. Researchers can easily access these data via the `recount` Bioconductor package, and can quickly import gene, exon, exon-exon junction and base-pair coverage data for uniformly processed data from the SRA, GTEx and TCGA projects in R for analysis. This workshop will cover different use cases of the `recount` package, including downloading and normalizing data, processing and cleaning relevant phenotype data, performing differential expression (DE) analyses, and creating reports for exploring the results using other Bioconductor packages. The workshop will also cover how to use the base-pair coverage data for annotation-agnostic DE analyses and for visualizing coverage data for features of interest. After taking this workshop, attendees will be ready to enhance their analyses by leveraging RNA-seq data from 70,000 human samples."

# Summary. An optional shortened abstract.
summary = "R/Bioconductor workshop on how to use recount2"

# Is this a featured talk? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["recount2"]

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Optional filename of your slides within your talk folder or a URL.
url_slides = "https://speakerdeck.com/lcolladotor/bioconductor-2017-recount-workshop"

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["recount2"]

# Links (optional).
url_pdf = "http://LieberInstitute.github.io/recountWorkshop/compiled_vignette/BioC2017.pdf"
url_video = ""
url_code = "http://LieberInstitute.github.io/recountWorkshop/"

# Demo talk page uses LaTeX math.
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Right"
+++

<script async class="speakerdeck-embed" data-id="7d2ab9486f174ef5a8a8134bf60cab74" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>


