+++
title = "Comprehensive assessment of multiple biases in small RNA sequencing reveals significant differences in the performance of widely used methods"
date = "2018-10-30"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Carrie Wright __*__", "Anandita Rajpurohit __*__", "Emily E. Burke"," Courtney Williams", "[__Leonardo Collado-Torres__](/authors/admin)", "Martha Kimos", "Nicholas J. Brandon", "Alan J. Cross", "Andrew E. Jaffe", "Daniel R. Weinberger  &dagger;", "Joo Heon Shin  &dagger;"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Preprint / Working Paper
# 4 = Report
# 5 = Book
# 6 = Book section
# 7 = Thesis
# 8 = Patent
publication_types = ["3"]

# Publication name and optional abbreviated version.
publication = "bioRxiv 445437"
publication_short = ""

# Abstract.
abstract = "High-throughput sequencing offers advantages over other quantification methods for microRNA (miRNA), yet numerous biases make reliable quantification challenging. Previous evaluations of reverse transcription or amplification bias in small RNA sequencing has been limited. Furthermore, little work has evaluated isomiR (miRNA isoforms) quantifications or the influence of starting amount on performance. We therefore evaluated quantifications of canonical miRNA and isomiRs using four library preparation kits, with various starting amounts (100ng to 2000ng), as well as quantifications following the removal of duplicate reads using unique molecular identifiers (UMIs) to mitigate reverse transcription and amplification biases. Randomized adapter and adapter-free methods mitigated bias; however, the adapter-free method was especially prone to false isomiR detection. We demonstrate that using UMIs improves accuracy and we provide a guide for input amounts to improve consistency. Our data show differences and limitations of current methods, thus raising concerns about the validity of quantification of miRNA and isomiRs."

# Summary. An optional shortened abstract.
summary = ""

# Digital Object Identifier (DOI)
doi = "10.1101/445437"

# Is this a featured publication? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["miRNA"]

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = [""]

# Links (optional).
url_pdf = "https://www.biorxiv.org/content/early/2018/10/30/445437.full.pdf"
url_preprint = "https://www.biorxiv.org/content/10.1101/445437v3"
url_code = "https://github.com/LieberInstitute/miRNA_Kit_Comparison"
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# links = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = true

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Image credit: [**bioRxiv**](https://doi.org/10.1101/445437)"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++

<!-- More detail can easily be written here using *Markdown* and $\rm \LaTeX$ math code. -->
