---
title: "Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex"
authors:
- "Kristen R Maynard __*__"
- "[__Leonardo Collado-Torres__](/authors/admin) __*__"
- "Lukas M. Weber"
- "Cedric Uytingco"
- "Brianna K. Barry"
- "Stephen R. Williams"
- "Joseph L. Catallini II"
- "Matthew N. Tran"
- "Zachary Besich"
- "Madhavi Tippani"
- "Jennifer Chew"
- "Yifeng Yin"
- "Joel E. Kleinman"
- "Thomas M. Hyde"
- "Nikhil Rao"
- "Stephanie C. Hicks"
- "Keri Martinowich &dagger;"
- "Andrew E Jaffe &dagger;"


date: "2020-02-28T00:00:00Z"
doi: "10.1101/2020.02.28.969931"

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: "*bioRxiv*"
publication_short: ""

abstract: We used the 10x Genomics Visium platform to define the spatial topography of gene expression in the six-layered human dorsolateral prefrontal cortex (DLPFC). We identified extensive layer-enriched expression signatures, and refined associations to previous laminar markers. We overlaid our laminar expression signatures onto large-scale single nuclei RNA sequencing data, enhancing spatial annotation of expression-driven clusters. By integrating neuropsychiatric disorder gene sets, we showed differential layer-enriched expression of genes associated with schizophrenia and autism spectrum disorder, highlighting the clinical relevance of spatially-defined expression. We then developed a data-driven framework to define unsupervised clusters in spatial transcriptomics data, which can be applied to other tissues or brain regions where morphological architecture is not as well-defined as cortical laminae. We lastly created a web application for the scientific community to explore these raw and summarized data to augment ongoing neuroscience and spatial transcriptomics research (http://research.libd.org/spatialLIBD).

# Summary. An optional shortened abstract.
summary:

tags:
- spatial
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: https://www.biorxiv.org/content/10.1101/2020.02.28.969931v1
url_code: https://github.com/LieberInstitute/spatialLIBD
url_dataset: https://github.com/LieberInstitute/HumanPilot
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**bioRxiv**](https://doi.org/10.1101/2020.02.28.969931)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

<!--

{{% alert note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /alert %}}

{{% alert note %}}
Click the *Slides* button above to demo Academic's Markdown slides feature.
{{% /alert %}}

Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
-->